javascript/javascript1/week3/js/main.js

```diff
@@ -0,0 +1,28 @@
+// Calculate the sum of all the squares
+
+let arrayOfNumbers = []
+
+function sumOfNumbers() {
+    for (let index = 0; index < 6; index++) {
+        let number = +prompt(`Enter number ${index + 1}: `);
+        arrayOfNumbers.push(number);
+    }
+
+    function sumOfEven() {
```

Comment:
can this function go outside and pass here with parameter (array) instead making nested function

Reply:
Yes I will do that in the coming projects.
Infact I followed the lecture and want to implement it.